topics/imaging/tutorials/multiplex-tissue-imaging-TMA/tutorial.md

```diff
@@ -0,0 +1,421 @@
+>    - {% icon param-file %} *"Select masks for the OME Tiff image (Optional)"*: Output of Mesmer (or other segmentation tool)
+>    - *"Whether to do phenotyping"*: `Yes`
+>        - *"Select an embedding algorithm for scatterplot"*: `UMAP`
+>        - *"Input phenotyping keys"*: `Multiple choices`
+>            - *"Select the key(s)"*: `phenotype`
+>
+{: .hands_on}
+
+# Next steps: Compositional and spatial analyses
+
+Galaxy-MTI includes additional tools from **Scimap** and tools from the **Squidpy** package ({% cite Palla2022 %}) that can be used to perform a variety of downstream analyses. For example, once phenotypes have been assigned to individual cells, **Squidpy** has several methods for understanding the spatial organization of the tissue. Using **Squidpy**, a spatial neighborhood graph is first generated, from which the organization of specific phenotype groups and their interactions can be quantified. 
+
+> ### {% icon hands_on %} Hands-on: Spatial analysis with **Squidpy**
+>
+> 1. {% tool **Squidpy Graph and Plotting** %} generate a spatial neighborhood graph with the following parameters:
+>
+>    - {% icon param-file %} *"Select the input anndata"*: Anndata file containing phenotype information (or other variable of interest)
+>    - *"Select an analysis"*: `Spatial neighbors -- Create a graph from spatial coordinates`
+>
+> 2. {% tool **Squidpy Graph and Plotting** %} compute and plot a neighborhood enrichment analysis with the following parameters:
+>
+>    - {% icon param-file %} *"Select the input anndata"*: Output of step 1 (anndata file with spatial neighborhood graph)
+>    - *"Select an analysis"*: `nhood_enrichment -- Compute neighborhood enrichment by permutation test`
+>    - *"Key in anndata.AnnData.obs where clustering is stored"*: `phenotype`
+>
+>    > ### {% icon comment %} Neighborhood enrichment plot
+>    >
+>    > **Squidpy** was used to calculate neighborhood enrichments for each phenotype in core 2 of exemplar 2 (Figure 10.). This shows which phenotypes co-locate most frequently within the tissue. 
+>    >
+>    > ![enrichment](../../images/multiplex-tissue-imaging-TMA/ex2_squidpy_enrichment.png "The output of Squidpy's neighborhood enrichment on core 2 from Exemplar-002.")
+>    >
+>    {: .comment}
+>
+> 3. {% tool **Squidpy Graph and Plotting** %} calculate Ripley's L curves for each phenotype with the following parameters:
+>
+>    - {% icon param-file %} *"Select the input anndata"*: Output of step 1 (anndata file with spatial neighborhood graph)
+>    - *"Select an analysis"*: `nhood_enrichment -- Compute neighborhood enrichment by permutation test`
+>    - *"Key in anndata.AnnData.obs where clustering is stored"*: `phenotype`
+>    - In *"Advanced Graph Options"*:
+>        - *"Which Ripley's statistic to compute"*: `L`
+>    - In *"Plotting Options"*:
+>        - *"Ripley's statistic to be plotted"*: `L`
```

Comment:
I'm afraid I don't see either of these options

![image](https://user-images.githubusercontent.com/458683/178721748-93f421a1-393e-43e7-b5c5-248681a447ab.png)